docs/src/developer/reference/config-options.md

```diff
@@ -463,6 +463,20 @@ federator:
     clientPrivateKey: client-key.pem
 ```
 
+## Outlook calalendar integration
+
+This feature setting only applies to the Outlook Calendar extension for Wire. As it is an external service, it should only be configured through this feature flag and otherwise ignored by the backend.
```

Comment:
`s/As it is an external service, it should only be configured through this feature flag and otherwise ignored by the backend./This does not change the behavior of the backend, but it will be consulted by the [calendar integration service](TODO: which repo?  ask dejan?), which will adopt its behavior accordingly./`